routes/api/asset/index.js

```diff
@@ -82,18 +82,11 @@ router.post('/:asset_id/scrape', (req, res, next) => {
 });
 
 router.put('/:asset_id/settings', (req, res, next) => {
-
   // Override the settings for the asset.
   Asset
     .overrideSettings(req.params.asset_id, req.body)
-    .then(() => {
-
-      res.status(204).end();
-    })
-    .catch((err) => {
-      next(err);
-    });
-
+    .then(({settings}) => res.status(200).send({settings}))
```

Comment:
Updates can not return a payload, this should be reverted to return 204 without a body.